exporter/prometheusremotewriteexporter/README.md

```diff
@@ -43,6 +43,8 @@ Example:
 exporters:
   prometheusremotewrite:
     endpoint: "http://some.url:9411/api/prom/push"
+    wal:
```

Comment:
Let's try to align with https://docs.google.com/document/d/1cCcoFgjDFwU2n823tKuMvrIhzHty4UDyn0IcfUHiyyI/edit#heading=h.mlf37ibqjgov.

Reply:
All done, thank you!

Reply:
Add a comment to explain what "wal" is.

Reply:
Done, thanks!